<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>importDialog</class>
 <widget class="QDialog" name="importDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import New Books</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string comment="A books title">Title</string>
       </property>
       <property name="buddy">
        <cstring>titleEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KLineEdit" name="titleEdit">
       <property name="toolTip">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string comment="A person who writes books.">Author</string>
       </property>
       <property name="buddy">
        <cstring>authorEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KLineEdit" name="authorEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string comment="A books edition or release number.">Release Number</string>
         </property>
         <property name="buddy">
          <cstring>relNumEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KLineEdit" name="relNumEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Release Date</string>
         </property>
         <property name="buddy">
          <cstring>relDateEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KLineEdit" name="relDateEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Series</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KLineEdit" name="seriesEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Volume</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KLineEdit" name="volumeEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string comment="A books type or style, examples- Science fiction or history">Genre</string>
       </property>
       <property name="buddy">
        <cstring>genreEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KLineEdit" name="genreEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string comment="The on disk path to a book.">Location</string>
       </property>
       <property name="buddy">
        <cstring>locationUrlRequestor</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KUrlRequester" name="locationUrlRequestor"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string comment="A brief summary or description of the book">Description</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KTextEdit" name="descEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="KPushButton" name="ImportButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>titleEdit</tabstop>
  <tabstop>authorEdit</tabstop>
  <tabstop>relNumEdit</tabstop>
  <tabstop>relDateEdit</tabstop>
  <tabstop>seriesEdit</tabstop>
  <tabstop>volumeEdit</tabstop>
  <tabstop>genreEdit</tabstop>
  <tabstop>locationUrlRequestor</tabstop>
  <tabstop>descEdit</tabstop>
  <tabstop>ImportButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>importDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
